proto/redpanda/api/console/v1alpha1/security.proto

```diff
@@ -229,7 +100,7 @@ service SecurityService {
   // ListRoleMembership lists all the members assigned to a role based on optional filter.
   rpc ListRoleMembers(ListRoleMembersRequest) returns (ListRoleMembersResponse) {
     option (redpanda.api.auth.v1.authorization) = {
-      required_permission: PERMISSION_ADMIN
+      required_permission: PERMISSION_VIEW
```

Comment:
Should `ListRoles`, `GetRole`, and `ListRoleMembers` be available for viewer permissions?